Packs/CommonTypes/Layouts/layout-edit-Access.json

```diff
@@ -1,186 +0,0 @@
-{
```

Comment:
Why did you remove this file, and not add a `toversion` ?

Reply:
didnt really remove it, just moved it to `AccessInvestigation` content pack and added `toVersion`
not sure why github shows it like that